policy/datasets.yaml

```diff
@@ -974,6 +982,14 @@ fgcmCalibrateTract_config:
     storage: ConfigStorage
     python: lsst.fgcmcal.fgcmCalibrateTract.FgcmCalibrateTractConfig
     template: config/fgcmCalibrateTract.py
+fgcmCalibrateTractTable_config:
+    description: >-
```

Comment:
Why the `>-` here and not in the other one?

Reply:
I dunno, that's what my editor (emacs) automatically put in there.  I guess it's not necessary?

Reply:
I don't think it's necessary: I don't see it in any of our other YAML files.